src/components/elements/annotationSidePanel/components/AnnotationsOverview.tsx

```diff
@@ -0,0 +1,111 @@
+/* Import Dependencies */
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { Formik, Form } from "formik";
+import { Row, Col, Card } from "react-bootstrap";
+
+/* Import Types */
+import { Annotation } from "app/types/Annotation";
+
+/* Import Icons */
+import { faGears, faPenToSquare } from "@fortawesome/free-solid-svg-icons";
+
+/* Import Components */
+import AnnotationCard from "./AnnotationCard";
+import SortingFilters from "./SortingFilters";
+import { Button } from "components/elements/customUI/CustomUI";
+
+
+/* Props Type */
+type Props = {
+    annotations: Annotation[],
+    StartAnnotationWizard: Function
+};
+
+
+/**
+ * Component that renders the annotations overview in the annotation side panel
+ * @param annotations The annotations to be rendered in the overview
+ * @param StartAnnotationWizard Function that starts the annotation wizard
+ * @returns JSX Component
+ */
+const AnnotationsOverview = (props: Props) => {
+    const { annotations, StartAnnotationWizard } = props;
+
+    /* Base variables */
+    const initialFormValues = {
+        filterByMotivation: '',
+        sortByDate: ''
+    };
+
+    return (
+        <div className="h-100 d-flex flex-column">
```

Comment:
can you use bootstrap instead of flexbox?

Reply:
These are bootstrap classes :D